Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/Unit.cs

```diff
@@ -0,0 +1,21 @@
+﻿using UnityEngine;
+using System.Collections;
+
+[System.Serializable]
+public class Unit : MonoBehaviour {
+
+	public static UnitBaseData UnitBaseData;
+	//TODO: Decide if this is the best way to handle an upgrade. Is it better to have a separate Upgrade class?
+	public UnitBaseData Upgrade;
+	public TileData Position;
+	public static UnitType Type;
+
+	void Start () {
+	
+	}
+
+
```

Comment:
Maybe we could do with getHP, getStrength etc... which add the upgrade+basedata together if upgrade exists?